rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -1659,4 +1660,11 @@ public ClearingRequest updateCRSize(ClearingRequest clearingRequest, Project pro
     public boolean isWriteActionAllowed(Object object, User user) {
         return makePermission(object, user).isActionAllowed(RequestedAction.WRITE);
     }
+
+    public void throwIfSecurityUser(User user) {
+        if (user.getUserGroup().name().equals("SECURITY_USER")) {
```

Comment:
A quick question, can we not use `PermissionUtils.isSecurityUser(user)` here?

Reply:
yes, we can. Thanks for pointing out.